src/Tickets/Commerce/Module.php

```diff
@@ -449,20 +449,43 @@ public function supports_global_stock() {
 	 * @return string
 	 */
 	public function get_price_html( $product, $attendee = false ) {
-		return tribe( Ticket::class )->get_price_html( $product, $attendee );
+		$value = $this->get_price_value( $product );
+
+		$price_html = '<span class="tribe-tickets-price-amount amount">' . esc_html( $value->get_currency() ) . '</span>';
+
+		/**
+		 * Allow filtering of the Price HTML
+		 *
+		 * @since TBD
+		 *
+		 * @param string $price_html
+		 * @param mixed  $product
+		 * @param mixed  $attendee
+		 *
+		 */
+		return apply_filters( 'tec_tickets_commerce_ticket_price_html', $price_html, $product, $attendee );
 	}
 
 	/**
-	 * Gets the product price value
+	 * Gets the product price value object
 	 *
 	 * @since 5.1.9
+	 * @since TBD method signature changed to return an instance of Value instead of a string.
+	 *
+	 * @version TBD
 	 *
 	 * @param int|\WP_Post $product
 	 *
-	 * @return string
+	 * @return Commerce\Utils\Value;
 	 */
 	public function get_price_value( $product ) {
-		return tribe( Ticket::class )->get_price_value( $product );
+		$ticket = Commerce\Models\Ticket_Model::from_post( $product );
+
+		if ( ! is_a( $ticket, 'TEC\Tickets\Commerce\Models\Ticket_Model' ) ) {
+			return;
+		}
+
+		return $ticket->get_value();
```

Comment:
We 100% want to avoid adding anything to the Module file as aligned originally on the Tickets Commerce document we are creating.

This needs to live in the Ticket Class.